modules/steam/autostart.nix

```diff
@@ -36,16 +36,16 @@ in
               lib.assertMsg (userProvidedDesktopSession != null -> (str.check userProvidedDesktopSession && lib.elem userProvidedDesktopSession config.services.xserver.displayManager.sessionData.sessionNames)) ''
                   Desktop session '${userProvidedDesktopSession}' not found.
                   Valid values for 'jovian.steam.desktopSession' are:
-                    ${lib.concatStringsSep "\n  " (lib.lists.remove "gamescope-wayland" config.services.xserver.displayManager.sessionData.sessionNames)}
-                  If you don't want a desktop session to switch to, remove 'jovian.steam.desktopSession' from your config.
+                    ${lib.concatStringsSep "\n  " config.services.xserver.displayManager.sessionData.sessionNames}
+                  If you don't want a desktop session to switch to, set 'jovian.steam.desktopSession' to 'gamescope-wayland'.
               '';
           };
           default = null;
           example = "plasma";
           description = lib.mdDoc ''
             The session to launch for Desktop Mode.
 
-            By default, attempting to switch to the desktop will launch
+            By default value of null or 'gamescope-wayland', attempt to switch to the desktop will launch
```

Comment:
I think we don't need to reword this? Switching to gamescope-wayland _is_ technically switching to another session that is just gaming mode again.